src/client.rs

```diff
@@ -1,107 +1,132 @@
 use base64::{engine::general_purpose, Engine as _};
+use bytes::Bytes;
+use etherparse::Ipv4Header;
+use futures::stream::SplitSink;
 use futures::stream::StreamExt;
```

Comment:
Let's merge `use` from the same lib with `::{}`

Reply:
Oh, I thought `cargo fmt` would fix that for me